Controller/Adminhtml/Bookmark/Index.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inchoo\ProductBookmark\Controller\Adminhtml\Bookmark;
+
+use Magento\Backend\App\Action;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\View\Result\Layout;
+
+class Index extends Action
+{
+
+    /**
+     * @return bool
+     */
+    protected function _isAllowed()
+    {
+        return $this->_authorization->isAllowed('Inchoo_ProductBookmark::productbookmark');
+    }
```

Comment:
po meni je ovako bolje:
```suggestion
    public const ADMIN_RESOURCE = 'Inchoo_ProductBookmark::productbookmark';
```

Reply:
Slazem se ! Prepravljeno